backend/layers/business/business.py

```diff
@@ -692,6 +693,20 @@ def tombstone_collection(self, collection_id: CollectionId) -> None:
         self.delete_all_dataset_versions_from_public_bucket_for_collection(collection_id)
         self.database_provider.tombstone_collection(collection_id)
 
+    def resurrect_collection(self, collection_id: CollectionId) -> None:
+        """
+        Resurrect a tombstoned Collection (untombstone Collection and un-delete public s3 assets). For admin use only.
+        """
+        collection = self.get_canonical_collection(collection_id)
+        if not collection.tombstoned:
+            raise CollectionIsNotTombstonedException()
+        all_dataset_versions = self.database_provider.get_all_dataset_versions_for_collection(collection_id)
+        for dv in all_dataset_versions:
+            for ext in (DatasetArtifactType.H5AD, DatasetArtifactType.RDS):
+                object_key = f"{dv.version_id}.{ext}"
+                self.s3_provider.resurrect_object(os.getenv("DATASETS_BUCKET"), object_key)
+        self.database_provider.untombstone_collection(collection_id)
```

Comment:
I would stick with untombstone or the resurrect notation. I can see this becoming a confusing topic differentiating between tombstone, deleted, untombstone, and resurrected,

Reply:
Sure. I was thinking that "resurrect" is the _business_ term while "tombstone" is the _persistence_ term. But I can use "resurrect" everywhere.